services/recipeservice/src/test/java/ch/whatshouldieat/services/controller/RecipeControllerTest.java

```diff
@@ -0,0 +1,52 @@
+package ch.whatshouldieat.services.controller;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+
+import ch.whatshouldieat.services.recipeservice.controller.RecipeController;
+import ch.whatshouldieat.services.recipeservice.model.Recipe;
+import ch.whatshouldieat.services.recipeservice.service.RecipeService;
+
+@SpringBootTest(webEnvironment=WebEnvironment.RANDOM_PORT)
+public class RecipeControllerTest {
+
+    @Autowired
+    RecipeService recipeService;
+
+    @Autowired
+    TestRestTemplate template;
+
+    public RecipeControllerTest(){
```

Comment:
Is this constructor really needed when empty?

Reply:
I added tests skeletons last week but have some issues with mocking the mongodb. As the existing code can not simply be tested as pure unit tests. I will have to put some research in on how to mock the db first. Tests are marked as @disabled for now so they should not break builds.